docs/sources/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1093,6 +1093,11 @@ Status Codes:
 -   **200** – no error
 -   **500** – server error
 
+> **Note**:
+> When the `remote` parameter references a Dockerfile then the `dockerfile`
+> parameter will have effect and the contents of the Dockerfile are placed
+> into a file called `Dockerfile`.
+
```

Comment:
@moxiegirl not sure why I had the 'note' down there, I moved it up to be after the parameters.
I was missing a "no" on "no effect" - maybe that'll help.

But the intent here is that "dockerfile" is used to tell the daemon where to find the Dockerfile within the build context. When "remote" is used and it points directly to a Dockerfile, the "dockerfile" parameter doesn't really mean anything because there is no Dockerfile in the context, its coming from some remote URL.  I'm open to a better wording of it.

re: git - the "remote" parameter can point to a git repo or a URL/file/dockerfile. When it points to a git repo though, the "dockerfile" parameter can still be specified because then it tells the daemon where in the repo to find the Dockerfile.